pkg/api/api.go

```diff
@@ -399,6 +399,10 @@ func (hs *HTTPServer) registerRoutes() {
 
 		// short urls
 		apiRoute.Post("/short-urls", bind(dtos.CreateShortURLCmd{}), routing.Wrap(hs.createShortURL))
+
+		if hs.Live.IsEnabled() {
+			apiRoute.Post("/live/telemetry/*", hs.LiveTelemetry.Handle)
```

Comment:
not very happy to pass channel path as part of API url here - looks like alternatives are custom header or url param.

Reply:
do we want "telemetry" as an additional scope?  if so, does this path conflict?

Reply:
At this moment everything published to this endpoint goes to `telemetry/{PATH}/{METRIC_NAME}` where `{PATH}` is `*` from route handler path `/live/telemetry/*`.

So:
* POST to `/live/telemetry/test` will result into publish into `telemetry/test/cpu`, `telemetry/test/system` 
* POST to `/live/telemetry/test/2` will result into publish into `telemetry/test/2/cpu`, `telemetry/test/2/system`

But it's not obvious to me whether this way of channel construction is ok or not.

Reply:
I thin we should require that the POST 'path' component does not contain a slash -- ie `POST to /live/telemetry/test/2` should be an error, but POST to /live/telemetry/test-2 is OK

Reply:
maybe:
```
                if hs.Live.IsEnabled() {
			apiRoute.Post("/live/push/:id", hs.LiveTelemetry.Handle)
		}
		```

where id has to be a simple string?